src/main/java/duke/pack/Task.java

```diff
@@ -0,0 +1,61 @@
+package duke.pack;
+
+/**
+ * represents a task of the user
+ */
+public class Task {
+    protected String description;
+    protected String fullDesc;
```

Comment:
child classes of task could inherit fullDesc instead of each child having it's own